apps/api/src/app/admin/admin.service.ts

```diff
@@ -45,6 +45,13 @@ export class AdminService {
     symbol
   }: UniqueAsset): Promise<SymbolProfile | never> {
```

Comment:
`UniqueAsset & { currency?: string }`

(use `currency: this.request.user.Settings.settings.baseCurrency` in `AdminController`)

Reply:
How to pass two set of arguments from `AdminController`?

AdminController:
```
return this.adminService.addAssetProfile({ dataSource, symbol,currency} );
  }
```

And to receive it at `AdminService`

```
  public async addAssetProfile({
    dataSource,
    symbol
  }: UniqueAsset  & { currency?: String })

```

Reply:
Yes, like this in `AdminController`:
```
this.adminService.addAssetProfile({ dataSource, symbol, currency: this.request.user.Settings.settings.baseCurrency } )
```

And `AdminService` like this:

```
public async addAssetProfile({
  currency,
  dataSource,
  symbol
}: UniqueAsset  & { currency?: String })
```

Reply:
This is throwing an error while passing `currency`:

<img width="655" alt="image" src="https://github.com/ghostfolio/ghostfolio/assets/77065548/42b997a6-e375-4b64-b023-c5e40c34c7e6">